lib/game.rb

```diff
@@ -2,44 +2,43 @@
 require "./lib/game_message"
 require "./lib/board"
 require "./lib/ship"
+require "./lib/player_setup"
 
 class Game
-  attr_reader :game_message,
-              :computer_board,
-              :player_board,
-              :board_setup,
-              :computer_cruiser,
-              :computer_sub,
-              :player_cruiser,
-              :player_sub
-
```

Comment:
Great job using what we discussed in check-in today and applying it directly to our code. This really helps me understand it further!